students/shibin_mathew/lesson2/series.py

```diff
@@ -0,0 +1,72 @@
+def fibonacci(n):
+	'''This function returns the nth value of the fibonacci seires'''
+	first = 0
+	second = 0
+	third = 0
+	listT= []
+	for i in range(0,100):
+		if i==0:
+			listT.append(i)
+			first = i
+
+		elif i==1:
+			listT.append(i)
+			second = i
+		else:
+			listT.append(first+second)
+			third = first
+			first = second
+			second = third+second
+
+	return listT[n-1]
+
+def lucas(n):
+	'''This function returns the nth value of the lucas seires'''
+	first = 0
+	second = 0
+	third = 0
+	listL = []
+	for i in range(0,100):
+		if i == 0:
+			listL.append(2) #first number in the lucas sequence is 2
+			first = 2
+		elif i == 1:
+			listL.append(1) #second number in the lucas sequence is 1
+			second = 1
+		else:
+			listL.append(first+second)
+			third = first
+			first = second 
+			second = third+second
+	return listL[n-1]
+
+
+
+def sum_series(n, firstPar = 0, secondPar = 1):
+	#Generatlized function that implements the logic behind the fibonacci sequence and the lucas sequence
+	first = 0 #initial value
+	second = 0 #initial value
+	third = 0 #initial value
+	listS = []
+	for i in range(0,100): #default parameter set to 100
```

Comment:
let's say our max is higher (4mil), how would you solve it without having to generate full sequence only to get let's say 5th element?